docs/src/main/sphinx/security/overview.rst

```diff
@@ -0,0 +1,150 @@
+=================
+Security overview
+=================
+
+After the initial :doc:`installation </installation>` of your cluster, security
+is the next major concern for successfully operating Trino. This overview
+provides an introduction to different aspects of configuring security for your
+Trino cluster.
+
+Aspects of configuring security
+-------------------------------
+
+The default installation of Trino has no security features enabled. Security
+can be enabled for different parts of the Trino architecture:
+
+* :ref:`security-client`
+* :ref:`security-inside-cluster`
+* :ref:`security-data-sources`
+
+Suggested configuration workflow
+--------------------------------
+
+To configure security for a new Trino cluster, follow this best practice
+order of steps. Do not skip or combine steps.
+
+#. **Enable** :doc:`HTTPS and TLS </security/tls>`
```

Comment:
Does this render nicely so it is obvious that the title is a link? If not maybe add a sentence below that points to the doc instead?

Reply:
Maybe after the list... 

More information is available in our guide for HTTPS/TLS configuration.

Reply:
![image](https://user-images.githubusercontent.com/5950884/113775437-b227b500-96f6-11eb-9976-b8d4978c4436.png)

Reply:
Except Dain wants consistency, so: HTTPS/TLS